src/views/Home/Charts/MonthlyChart/index.tsx

```diff
@@ -0,0 +1,189 @@
+import { useMemo, useState } from 'react';
+import { encodeDate, listToMap } from '@togglecorp/fujs';
+import { useRequest } from '#utils/restRequest';
+
+import BlockLoading from '#components/BlockLoading';
+import Container from '#components/Container';
+import Button from '#components/Button';
+import { getDatesSeparatedByMonths } from '#utils/chart';
+import { resolveToComponent } from '#utils/translation';
+import useTranslation from '#hooks/useTranslation';
+
+import TimelineChart from '../TimelineChart';
+import PointDetails from '../PointDetails';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+const APPEAL_TYPE_EMERGENCY = 1;
+const APPEAL_TYPE_DREF = 0;
+
+const dateFormatter = new Intl.DateTimeFormat(
+    undefined,
+    {
+        month: 'long',
+    },
+);
+
+const getFormattedKey = (dateFromProps: string | Date) => {
+    const date = new Date(dateFromProps);
+    return `${date.getFullYear()}-${date.getMonth()}`;
+};
+
+interface AggregateResponse {
+    aggregate: {
+        timespan: string;
+        count: number;
+        amount_funded: string;
+        beneficiaries: number;
+    }[];
+}
+
+interface Props {
+    year: number;
+    onBackButtonClick: (year: undefined) => void;
+}
+
+function MonthlyChart(props: Props) {
+    const {
+        year,
+        onBackButtonClick,
+    } = props;
+    const strings = useTranslation(i18n);
+    const dateList = useMemo(
+        () => {
+            const startDate = new Date(year, 0, 1);
+            const endDate = new Date(year, 11, 31);
+            return getDatesSeparatedByMonths(startDate, endDate);
+        },
+        [year],
+    );
+
+    const [activePointKey, setActivePointKey] = useState<string>(
+        getFormattedKey(dateList[0]),
+    );
+
+    const queryParams = {
+        model_type: 'appeal',
+        start_date: encodeDate(new Date(year, 0, 1)),
+        end_date: encodeDate(new Date(year, 11, 31)),
```

Comment:
Maybe we should write helpers for this? It's always confusing to use `0` for first month